src/cow-react/modules/limitOrders/containers/LimitOrdersWidget/index.tsx

```diff
@@ -1,17 +1,31 @@
 import * as styledEl from './styled'
```

Comment:
So you decided to create a new module `limitOrders` for me limit orders and swap should be in the same module. But for now ignore I expect a lot of things to move around.

Reply:
Thank you!
Already discussed, will move it after PR merge (because there are other PR's related to the current structure)